Makefile

```diff
@@ -62,9 +62,9 @@ DEPINCLUDES=$(INCLUDES)
 OCAMLDOC_OPT=$(WITH_OCAMLDOC:=.opt)
 OCAMLTEST_OPT=$(WITH_OCAMLTEST:=.opt)
 
-BYTESTART=driver/main.cmo
+BYTESTART=driver/mainentry.cmo
 
-OPTSTART=driver/optmain.cmo
+OPTSTART=driver/optmainentry.cmo
```

Comment:
Thinking slightly of downstream packagers, couldn't this be done the other way around - so the bulk of code is in `mainentry.ml` and `optmainentry.ml` (or possible `maindriver.ml`, `optmaindriver.ml`?) and it remains the case that the extra installed modules are `main` and `optmain`?

Reply:
Good idea, changed.